packages/components/src/foundations/breakpoints.ts

```diff
@@ -0,0 +1,12 @@
+// todo: understand if we can generate this list reading the correspondin Sass file
+// see: https://sergiocarracedo.es/2020/07/17/sharing-variables-between-scss-and-typescript/
+
+export const hdsBreakpointsValues = {
```

Comment:
in order to follow ember conventions this would be in utils folder

Reply:
additionally why we need this file? not everything needs to be a component or helper we can just use css

Reply:
also something like this should be possible as well https://medium.com/@miroslav_petrov/how-to-share-variables-between-scss-and-js-7b4225830abe

Reply:
> not everything needs to be a component or helper we can just use css

It will be needed in our JS implementation ([for the AppSideNav collapsing behaviour](https://github.com/hashicorp/design-system/blob/main/packages/components/src/components/hds/app-side-nav/index.ts#L36-L38)) and potentially/likely also consumers may need to access some of these values in JS if they need to implement logic related to media-queries

Reply:
I would lean towards keeping this manual and it changes a lot on regular bases we can look into automatic creation of this map from sass

Reply:
> also something like this should be possible as well if you really need this file https://medium.com/@miroslav_petrov/how-to-share-variables-between-scss-and-js-7b4225830abe

yes, see my comment at line 2 in this file.

in any case I tried, and I didn't manage to get it working; it could not find the file (I used different paths, local/relative or the package path), and also TS was complaining about not using a mixin
<img width="822" alt="modified" src="https://github.com/user-attachments/assets/42a7e0fb-4bcb-4520-9c98-0f6d9a3c6ffe" />

So I don't think is worth to spend too much time on this, it's a nice to have but low priority at the moment